packages/renderer/src/lib/dialogs/QuickPickInput.spec.ts

```diff
@@ -27,19 +27,18 @@ import { beforeAll, describe, expect, test, vi } from 'vitest';
 import type { InputBoxOptions, QuickPickOptions } from './quickpick-input';
 import QuickPickInput from './QuickPickInput.svelte';
 
-const sendShowQuickPickValuesMock = vi.fn();
-const sendShowInputBoxValueMock = vi.fn();
-const sendShowQuickPickOnSelectMock = vi.fn();
 const receiveFunctionMock = vi.fn();
+const sendShowQuickPickOnSelectMock = vi.fn().mockResolvedValue(undefined);
+const sendShowQuickPickValuesMock = vi.fn().mockResolvedValue(undefined);
```

Comment:
should be removed

Reply:
Still needed (see other comment), in order for this to work correctly